### ISSUE TITLE
Unable to use custom fonts

### ISSUE THREAD

**Issue by metasoarous** [gered/clj-htmltopdf#6]
Hi; Thanks for putting out this project.

I'm having trouble getting custom fonts to render. Should something like this work?

```clj
(pdf/->pdf
  [:div
   {:style {:font-family "concrete-roman"}}
   [:h1 "HTML to PDF"]
   [:p "With glorious concrete roman!"]]
  "test.pdf"
  {:styles {
            :styles ["/home/csmall/code/oz/public/css/style.css"]
            :fonts [{:font-family "concrete-roman"
                     :src "/home/csmall/code/oz/public/fonts/computer-modern/cmunorm.ttf"}
                    {:font-family "concrete-roman"
                     :src "/home/csmall/code/oz/public/fonts/computer-modern/cmunoti.ttf"}
                    {:font-family "concrete-roman"
                     :src "/home/csmall/code/oz/public/fonts/computer-modern/cmunobx.ttf"}
                    {:font-family "concrete-roman"
                     :src "/home/csmall/code/oz/public/fonts/computer-modern/cmunobi.ttf"}]}})
```

Everything renders, but it's just using some random serif font instead. Am I missing something?

Thanks very much.

**Comment by gered** [comment]
Hello,

Took me a little bit to find this, but what finally highlighted the problem was adding this bit to the `->pdf` function's third argument (the options map):

```clojure
:debug  {:display-html?    true
         :display-options? true}
```

That showed me that the `<div>` tag was being rendered like so:

```html
<div style="{:font-family &quot;concrete-roman&quot;}">
```

Which is obviously wrong. :-)

clj-htmltopdf just utilizes Hiccup itself for rendering HTML in Clojure/EDN form as you are doing, and hiccup doesn't know how to convert EDN-style CSS to a CSS string last I checked?

Anyway, the easiest solution for you right now is to use an actual CSS style string for inline styles like this, e.g.

```clojure
[:div
 {:style "font-family: concrete-roman"}
 [:h1 "HTML to PDF"]
 [:p "With glorious concrete roman!"]]
```

After I made that change using your example, the PDF was rendered using the custom font as expected.

Hope this helps!
